app/src/test/java/org/simple/clinic/drugs/selection/PrescribedDrugsScreenControllerTest.kt

```diff
@@ -57,15 +59,16 @@ class PrescribedDrugsScreenControllerTest {
   private lateinit var fixture: MobiusTestFixture<EditMedicinesModel, EditMedicinesEvent, EditMedicinesEffect>
 
   @Before
-  fun setup() {
+  fun instantiateFixture() {
```

Comment:
Can we continue calling this `setup()`? This is the convention for the JUnit4 test fixtures and I think it's fine to keep it that way.